pilot/pkg/config/kube/gateway/model.go

```diff
@@ -35,6 +37,16 @@ const (
 	serviceTypeOverride          = "networking.istio.io/service-type"
 )
 
+var supportedFeatures = suite.AllFeatures
+
+func getSupportedFeatures() []k8sv1.SupportedFeature {
```

Comment:
nit: you can make this a static var instead of computing it each time `var supported = func() []k8sv1.SupportedFeature { ... }()`

Reply:
changed thanks